Parser/convertSBEcnvVar.m

```diff
@@ -137,9 +140,11 @@
                 comment = 'PAR/Logarithmic/ Satlantic';
 
                 % artificial chlorophyll from fluorescence (mg/m3)
+                % SBE manual indicate this is a Wet Labs - ECO AFL sensor
             case 'flECO0x2DAFL'
                 name = 'CPHL';
-                comment = 'Artificial chlorophyll data computed from bio-optical sensor raw counts measurements.';
```

Comment:
In this case for example the unit is already in the right form. It wasn't in ug/l. Please re-check across all parsers.

Reply:
According to the SBE manual, flECOAFL is outputed in mg/m3 which is ug/l, so it's correct.